### PR TITLE
Switched the dropdown arrow

### LINKED ISSUE #1242: Change arrow on sorting dropdown
https://app.zeplin.io/project/5acbd659dc553ec84805afd4/screen/5b45ad8f22c744f901194d96

### PR DESCRIPTION
**Pull request by smcmurtry**
Closes #1242

### CODE REVIEW

File: components/dropdown.js [hunk 1]
```diff
@@ -19,9 +21,9 @@ const right = css`
 const selectStyle = css`
   -webkit-appearance: none;
   -moz-appearance: none;
-  background-position: right 50%;
+  background-image: url('data:image/svg+xml;utf-8,<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><path d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z" fill="${cerulean}"/><path d="M0 0h24v24H0z" fill="none"/></svg>');
```
Comment (sastels) [inline components/dropdown.js:24]: How does this work, are we downloading something from the web here?
Why not just use the `ExpandMoreIcon` as we do in the card footer?

Comment (smcmurtry) [reply components/dropdown.js:24]: My fuzzy understanding is that you always include background images in css with this url function. If the image is hosted somewhere you pass in that url, if not you can just put the image source code in (which is what I've done here).

I believe the `xmlns="http://www.w3.org/2000/svg"` doesn't download anything - it just indicates what version of the svg specification we're using.

Comment (sastels) [reply components/dropdown.js:24]: ah, I see. interesting. 👍 
Should we replace our material ui `ExpandMoreIcon` with this one?

**Review by sastels — APPROVED** [review]
👍 cool!

**Pull request merged**